internal/commands/pull/command.go

```diff
@@ -258,17 +258,22 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
-	successfulTemplateWrites := make([]string, 0, len(clientTemplates))
+	successfulTemplateWrites := make([]interface{}, 0, len(clientTemplates))
 	for _, ct := range clientTemplates {
 		if err := local.WriteZip(filepath.Join(pathFrontend, ct.id), ct.zipPkg); err != nil {
 			return fmt.Errorf("unable to save template '%s' to disk: %s", ct.id, err)
 		}
-		successfulTemplateWrites = append(successfulTemplateWrites, ct.id)
+		readmeFile := ""
+		if matches, err := filepath.Glob(filepath.Join(pathRelative, local.FrontendPath, ct.id, "README.*")); err == nil && len(matches) == 1 {
+			readmeFile = matches[0]
+		}
+		successfulTemplateWrites = append(successfulTemplateWrites, fmt.Sprintf("[%s] %s", ct.id, readmeFile))
```

Comment:
my guess is the original `[%s]` style formatting was a mistake (of using `[]string` instead of spreading `[]interface{}` as the last arg to `NewListLog`.  do we still want to preserve that?

seems like something a little less distracting could just be:

`fmt.Sprintf("%s: %s", ct.id, readmeFile)`

again, this is just another thing we should run by design... it may help to schedule a quick 15 min screen share with them so they can really see the output in action